README.md

```diff
@@ -6,20 +6,30 @@ STIG-Partitioned Enterprise Linux (_spel_) is a project that helps create and
 publish Enterprise Linux images that are partitioned according to the
 [DISA STIG][0]. The resulting images also use LVM to simplify volume management.
 The images are configured with help from the scripts and packages in the
-[`AMIgen7`][31], and [`AMIgen8`][40] projects.
+[`AMIgen7`][31], and [`AMIgen8`][40] projects[^1].
 
 ## Why spel
 
 VMs' root filesystems are generally not live-repartitionable once launced from
 their images. As a result, if a STIG-scan is performed against most of the
-community-published images for Red Hat and CentOS, those scans will note
-failures for each of the various "`${DIRECTORY}` is on its own filesystem"
-tests. The images produced through this project are designed to ensure that
-these particular scan-failures do not occur.
+community-published images for Red Hat and related distros (CentOS/CentOS
+Stream, [Oracle Linux][41], [Rocky][42], [Alma][43] or [Liberty][44]), those
+scans will note failures for each of the various "`${DIRECTORY}` is on its own
+filesystem" tests. The images produced through this project are designed to
+ensure that these particular scan-failures do not occur.
 
 Aside from addressing the previously-noted partitioning findings, spel does
-_not_ apply any STIG-related hardening. The spel-produced images are expected
-to act as a better starting-point in a larger hardening process.
+applies only those STIG-related hardenings that need to be in place "from
```

Comment:
The prior line ends with `does`, which makes this `does applies`. Maybe just delete `does`?

Reply:
Done

Reply:
Updated the block because I couldn't parse what you were saying versus how the GitHub conversation-view was displaying things.